new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorTwoDomains.java

```diff
@@ -187,26 +254,289 @@ public void testTwoDomainsManagedByTwoOperators() {
     createTwoDomainsOnPVUsingWlstAndVerify();
 
     // get the domain1 and domain2 pods original creation timestamps
-    getBothDomainsPodsOriginalCreationTimestamp();
+    getBothDomainsPodsOriginalCreationTimestamp(domainUids, domainNamespaces);
 
     // scale cluster in domain1 from 2 to 3 servers and verify no impact on domain2
     replicasAfterScale = 3;
     scaleDomain1AndVerifyNoImpactOnDomain2();
 
     // restart domain1 and verify no impact on domain2
-    restartDomain1AndVerifyNoImpactOnDomain2();
+    restartDomain1AndVerifyNoImpactOnDomain2(replicasAfterScale, domain1Namespace, domain2Namespace);
 
     // shutdown both domains and verify the pods were shutdown
-    shutdownBothDomainsAndVerify();
+    shutdownBothDomainsAndVerify(domainNamespaces);
+  }
+
+  /**
+   * Create one operator if it is not running.
+   * Create domain domain1 and domain2 dynamic cluster in default namespace, managed by operator1.
+   * Both domains share one PV.
+   * Verify scaling for domain2 cluster from 2 to 3 servers and back to 2, plus verify no impact on domain1.
+   * Shut down domain1 and back up, plus verify no impact on domain2.
+   * shutdown both domains
+   */
+  @Order(2)
+  @Test
+  public void testTwoDomainsManagedByOneOperatorSharingPV() {
+    // create two domains sharing one PV in default namespace
+    createTwoDomainsSharingPVUsingWlstAndVerify();
+
+    // get the domain1 and domain2 pods original creation timestamps
+    List<String> domainNamespaces = new ArrayList<>();
+    domainNamespaces.add(defaultNamespace);
+    domainNamespaces.add(defaultNamespace);
+    getBothDomainsPodsOriginalCreationTimestamp(domainUids, domainNamespaces);
+
+    // scale domain2 to 3 servers and back to 2 and verify no impact on domain1
+    scaleDomain2AndVerifyNoImpactOnDomain1();
+
+    // restart domain1 and verify no impact on domain2
+    restartDomain1AndVerifyNoImpactOnDomain2(replicaCount, defaultNamespace, defaultNamespace);
+
+    // shutdown both domains
+    //shutdownBothDomainsAndVerify(domainNamespaces);
+  }
+
+  @Order(3)
+  @Test
+  public void testDeployAppAndInstallIngressControllers() {
```

Comment:
add javadoc

Reply:
how about changing the class name to ItLoadBalancerTwoDomains or ItTwoDomainsLoadBalancer or some such to indicate that this class contains load balancer tests as well